### PR TITLE
DF-1440-fix-expandable-layout

### PR DESCRIPTION
**Pull request by jimmynotjim**
Fixed the multi-line layout in the contact list on the contact-us page
 
## Additions
 
None
 
## Removals
 
None
 
## Changes
 
- Removed left and right modifiers to stop the label and link from floating
- Added padding to the expandable header to keep inner text from running into the link
- Absolutely positioned the link to the reserved space created above
 
## Testing
 
Navigate to:

- /contact-us/
 
## Review
 
- @sebworks 
- @anselmbradford 
- @Scotchester 
 
## Preview
 
screen shot 2015-03-13 at 10 52 30 am

## Notes
 
- Expandables are used throughout the site, further modifications need to be tested at `/the-bureau/bureau-structure/`
- Expandables could use a wider cleanup, but it's outside the scope of this task.
- Left a note in cf-enhancements stating the same

**Ignore the merge conflict, I'll fix this right after I get a sign-off**

### CODE REVIEW

**Comment by Scotchester** [comment]
In my opinion, the "Show" links should be vertically positioned on the top line of a multiline item.

File: static/css/cf-enhancements.less [hunk 1]
```diff
@@ -500,3 +500,18 @@ tbody {
...
     eof: true
 */
+
+/**
+ * This is a temporary hack to fix the contact expandables. CF Expandables could use a
+ * larger rework, especially removing duplicated blocks (open/close states, double search
+ * filter buttons based on device size, etc)
+ */
+.expandable__padded .expandable_header {
+  padding-right: 90px;
```
Comment (Scotchester) [inline static/css/cf-enhancements.less]: Seems a little generous. The links appear to hover around 50px in width. I don't think we'd want to give more than 30px between them and the heading.

Comment (jimmynotjim) [inline static/css/cf-enhancements.less]: It's not that generous, it's only 20px 

screen shot 2015-03-12 at 5 09 18 pm

Worth noting that the 90px includes the 16px that already existed.

Comment (Scotchester) [inline static/css/cf-enhancements.less]: Ahhhh, right. I mistakenly assumed you were adding the padding to the `expandable_label`.

**Comment by Scotchester** [comment]
I found an example where this is going to cause a problem: http://[pick-a-demo-server]/the-bureau/history/

**Comment by jimmynotjim** [comment]
@Scotchester They're different font-sizes, do you think they should they be vertically centered, align with the cap line or align with the base line with the first line?

Thanks for finding that page, I hadn't noticed it was different when reviewing it. I think I have a solution already.

**Comment by Scotchester** [comment]
I think I would try aligned with the baseline first. If that looks weird, then I'd say vertically centered.

**Comment by jimmynotjim** [comment]
Ok, new solution keeping the floats and using widths. The history `h3` is inside the `header-left` modifier, so the history page is still broken, but I think if we move that around it'll fix that issue if everyone seems ok with this solution.

**Comment by sebworks** [comment]
:+1:

**Pull request merged**